.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -8,8 +8,15 @@ body:
     attributes:
       label: Description
       description:
-        Write a brief description of the question or issue, also include what you tried and what didn't work. *Please add screenshots if applicable along with* ***steps to reproduce the issue and reference links (if applicable)***. If you need help or if you want to to discuss something with us, join the WalkingPal [Discord community](https://discord.gg/Et3aQTsW6a)!
-
+        Write a brief description of the question or issue, also include what you tried and what didn't work. *Please add screenshots if applicable along with* ***steps to reproduce the issue and reference links (if applicable)***. If you need help or if you want to to discuss something with us, join the WalkingPal [Discord community](https://discord.gg/Et3aQTsW6a)!       
         You can assign this issue to yourself if you want to work on it.
     validations:
       required: true
+
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
```

Comment:
Remove **all** code of conduct labels

no such labels needed for the checkbox. Rest LGTM

Reply:
Also there is a `.idea` folder in you PR. Please Remove that folder